pkg/tenant/members.go

```diff
@@ -67,21 +98,29 @@ func (s *Server) TenantMemberCreate(c *gin.Context) {
 	c.JSON(http.StatusCreated, member)
 }
 
+// MemberList retrieves all members assigned to a tenant
```

Comment:
I'm not sure I'm quite understanding how the `MemberList` function differs from `TenantMemberList` above -- it looks like both are designed to extract members associated with a tenant from the database. Is the only difference the route?

Reply:
Yes, currently the difference between the two is that the route is different. If we were to assign a member to an organization, `MemberList` could be updated and then provide a distinction between the two.

Reply:
There's been a bit of confusion on my side about this as well, so I want to follow up after looking back at a discussion about the difference. `MemberList` will list members for an organization and `TenantMemberList` will list members for a tenant. The confusing part for me came about because the Member model has a tenant ID instead of an object ID. However, the tenant ID will also be stored on the organization so that will aid in making that distinction. I'll update the docstring.